index.html

```diff
@@ -0,0 +1,123 @@
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Reading Materials
+        </div>
+        <div class="col-6 col-md-12">
+          Offer references for reading materials to give context to your studies and to allow you to dig deep on your lessons.
+        </div>
+      </div>
+      <div class="row flex-md-column row2">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/play.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Supporting Contents
+        </div>
+        <div class="col-6 col-md-12">
+          To aid your studies, we have compiled a list of tutorials which we deem best from the web.
+        </div>
+      </div>
+       <div class="row flex-md-column row3">
+        <div class="col-2 col-md-12 my-auto d-flex justify-content-center">
+          <img src="images/terminal.svg" width="48" height="48" alt="video">
+        </div>
+        <div class="col-4 col-md-12 my-auto">
+          Programming Tasks
+        </div>
+        <div class="col-6 col-md-12">
+          Coding tasks, mainly javascript function coding, along with the associative hints and answers.
+        </div>
+      </div>
+    </div>
+    <button type="button" class="sub m-3 mb-5 btn getstarted">Get started with the coding Challenges</button>
+  </section>
+  <section class="about" id="about">
+    <div class="mainImg">
+      <h3 class="motto d-flex justify-content-center text-center pt-5">"Preparing you for the language of the future!"</h3>
+      <h1 class="description-header d-flex justify-content-center text-center pt-3">CODING CHALLENGES</h1>
+      <div class="about-description text-center">
+        <p class="pt-2">Coding challenges is designed to test your knowledge on programming languages and give context on how well you are performing. In the future, leaderboard will be integrated to track your performance compared to other programmers.</p>
+      </div>
+      <div class="about-passage lato text-center">
+        Please contact us for any further questions about CC <br>
+        <a class="text-decoration-none fw-bold text-dark" href="#">debmirmij@gmail.com</a>
+      </div>
+    </div>
+    <div class="d-flex flex-column justify-content-center">
+      <h1 class="coding-header mt-5">Coding Challenges Logo</h1>
+      <div class="d-flex justify-content-center">
+        <img src="images/cc-logo-large.png" width="400" alt="large logo">
+      </div>
+    </div>
+    <div class="partner mt-5 pb-4 d-flex flex-column justify-content-evenly">
+      <h1 class="coding-header partner-header mt-5">Partner</h1>
+      <div class="d-flex align-items-center justify-content-evenly"><a href="https://www.microverse.org/"><img src="images/microverse-transparent.png" width="150" alt="microverse"></a><a href="https://github.com/"><img src="images/github-transparent.png" width="130" alt="github"></a></div>
+    </div>
```

Comment:
- [ ] [REQUIRED] The `About section` should be created in a separate .html file and when the About section is clicked from the navigation bar, the `About section` should be displayed on a separate page. Please revise the About section design too.